Makefile

```
@@ -1,10 +1,12 @@
 UV     := uv --directory kmir
```

(while at it) we should use `--project` here
```suggestion
UV     := uv --project kmir
```

---

Done.